src/components/TodoCard/TodoCard.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+//style
+import './TodoCard.css'
+
+function TodoCard({ todo, ...props }) {
+    return (
+        <div className='todo-card'>
+            <div>ID : {todo?.id}</div>
+            <p className='todo-card--text'>{todo?.text}</p>
+            <div>Is important : {todo?.isImportant.toString()}</div>
```

Comment:
let's show 'Yes' or 'No' text instead of 'true' and 'false' value.
Keep boolean values under the hood